management/status_checks.py

```diff
@@ -660,7 +660,16 @@ def check_mail_domain(domain, env, output):
 			else:
 				output.print_error("MTA-STS policy is present but has unexpected settings. [{}]".format(policy[1]))
 		else:
-			output.print_error("MTA-STS policy is missing: {}".format(valid))
+			certmessage = ''
+			cert = get_ssl_certificates(env).get(domain)
+			if not cert:
+				certmessage = "No TLS certificate provisioned for this domain"
+			else:
+				cert_status = check_certificate(domain, cert['certificate'], cert['private-key'])
+				if cert_status[0] != 'OK':
+					certmessage = "TLS certificate is not valid"
```

Comment:
I'm suggesting having no message (no call to output.print_error) in these two conditions.

Reply:
Do you want to remove these text messages completely or print them not as an error?

Reply:
Remove completely. Sorry for the ambiguity.

Reply:
Feel free to edit this merge request as is suitable for the codebase. I'm no longer use MIAB actively and can't test my edit right now.

Reply:
Works for me 👍